forge-game/src/main/java/forge/game/ability/effects/RollPlanarDiceEffect.java

```diff
@@ -19,17 +19,19 @@ public class RollPlanarDiceEffect extends SpellAbilityEffect {
      */
     @Override
     public void resolve(SpellAbility sa) {
-        boolean countedTowardsCost = !sa.hasParam("NotCountedTowardsCost");
         final Player activator = sa.getActivatingPlayer();
         final Game game = activator.getGame();
 
-        if(countedTowardsCost) {
+        if (game.getActivePlanes() == null) { // not a planechase game, nothing happens
+            return;
+        }
+        if (!sa.hasParam("NotCountedForPDCost")) {
```

Comment:
hm i don't like that Param, can't we just get with a better way which ones should count, and which should not?

it is just for the Static Action, wouldn't it be better to mark the static action instead?